build.gradle

```diff
@@ -29,7 +29,8 @@ dependencies {
   implementation "org.springframework.boot:spring-boot-starter-actuator"
   implementation "org.springframework.boot:spring-boot-starter-validation"
   implementation "org.springframework.boot:spring-boot-starter-web"
-  testImplementation("org.springframework.boot:spring-boot-starter-test") {
+    implementation 'org.jetbrains:annotations:20.1.0'
```

Comment:
Could we remove this new dependency? I don't think its needed.

Reply:
yep, I will remove it.